tests/example_model/model.py

```diff
@@ -25,9 +26,9 @@ def _convert_to_pil(array):
     return Image.fromarray(array)
 
 
-class ExampleClassificationModel(ClassificationModelBase):
-    def __init__(self, model_path: str):
-        super().__init__(classes=CLASSES, model_path=model_path)
+class ClassificationModel(ClassificationModelBase):
+    def __init__(self):
+        super().__init__(classes=CLASSES, model_path=MODEL_PATH)
```

Comment:
We might need to bring experts' attention to the fact that they mustn't change the class name.

Reply:
I have added this to the template model in `telesto-models`.